frontend/graphql/feed.graphql

```diff
@@ -3,10 +3,24 @@ query feed {
     id
     title
     body
+    excerpt
+    readTime
+    createdAt
+    images {
+      smallSize
+    }
+    likes {
+      id
+    }
+    threads {
+      id
+    }
     author {
       id
+      handle
       name
       email
+      profileImage
     }
   }
 }
```

Comment:
Ideally we should try to use the fragments, you did update one in `fragments.graphql` but it looks like not using it anywhere here